modules/exploits/user_access_manager_reflected_xss_shell_upload.rb

```diff
@@ -0,0 +1,32 @@
+class Wpxf::Exploit::UserAccessManagerReflectedXssShellUpload < Wpxf::Module
+  include Wpxf::WordPress::ReflectedXss
+
+  def initialize
+    super
+
+    update_info(
+      name: 'User Access Manager <= 2.0.0 Reflected XSS Shell Upload',
+      author: [
+        'Paul Williams <phyushin[at]phyubox.com>' # WPXF module
+      ],
+      references: [
+        ['WPVDB', '8814'],
+        ['URL', 'http://www.defensecode.com/advisories/DC-2017-01-021_WordPress_User_Access_Manager_Plugin_Advisory.pdf'],
+        ['URL', 'http://seclists.org/bugtraq/2017/May/31']
+      ],
+      date: 'May 11 2017'
+    )
+  end
+
+  def check
+    check_plugin_version_from_readme('user-access-manager', '2.0.0')
```

Comment:
it is fixed in v 2.0.0 as i did try that just says "you're not allowed to access that page"